cmd/delete.go

```diff
@@ -101,8 +102,35 @@ func RunDelete(options *DeleteOptions) error {
 		return fmt.Errorf("Unable to init reconciler: %v", err)
 	}
 
-	if err := reconciler.Destroy(); err != nil {
-		return fmt.Errorf("Unable to destroy resources for cluster [%s]: %v", options.Name, err)
+	done := make(chan bool)
+	message := make(chan error)
+
+	go func() {
+		message <- reconciler.Destroy()
+	}()
+
+	go func(description string, symbol string, c chan bool) {
+		if description != "" {
+			fmt.Print(description)
```

Comment:
Can we use `logger.Info` instead of `fmt.Print`? It's soops important all of our output goes through the same place so we can manage things globally.